.github/workflows/integration-test.yml

```diff
@@ -13,36 +13,15 @@ jobs:
     name: Run integration tests
     runs-on: ubuntu-latest
     steps:
-      # build backend plugin
-      - name: Set up JDK 14
-        uses: actions/setup-java@v1
-        with:
-          java-version: 14.0.x
-
-      - name: Cache Maven packages
-        uses: actions/cache@v1
-        with:
-          path: ~/.m2
-          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
-          restore-keys: ${{ runner.os }}-m2
-
-      - name: Checkout security plugin
-        uses: actions/checkout@v2
-        with:
-          repository: 'opensearch-project/security'
-          path: security
-          ref: 'main'
-
-      - name: Build security plugin
-        working-directory: ./security
-        run: mvn -B clean package -Padvanced -DskipTests
-
       - name: Download OpenSearch Core
         run: |
-          wget https://artifacts.opensearch.org/snapshots/core/opensearch/1.2.0-SNAPSHOT/opensearch-min-1.2.0-SNAPSHOT-linux-x64-latest.tar.gz
+          wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.0/488/linux/x64/builds/opensearch/dist/opensearch-min-1.3.0-linux-x64.tar.gz
```

Comment:
This is a temporary fix, moving forward we want to point to a symlinked `latest` build of each `SNAPSHOT` version, after they are provided [here](https://github.com/opensearch-project/opensearch-build/pull/1378)

Reply:
It looks like your integration tests are running docker, can you pick up the latest docker opensearch distribution image as starting point instead - https://hub.docker.com/r/opensearchstaging/opensearch/tags?page=1&name=1.3.0?

I think this would clean up this integration test code sustainably by removing the interaction with build artifacts.